.github/workflows/commit-message-check.yaml

```diff
@@ -18,24 +18,26 @@ jobs:
     name: Commit Message Check
     steps:
     - name: Get PR Commits
+      if: ${{ !contains(github.event.pull_request.labels.*.name, 'force-skip-ci') }}
       id: 'get-pr-commits'
       uses: tim-actions/get-pr-commits@v1.0.0
```

Comment:
Not related with this PR. I am just curious why we are using that ^ action instead of the [`checommits`](https://github.com/kata-containers/tests/tree/main/cmd/checkcommits) program?  <---- @jodh-intel

Reply:
I have no idea, sincerely, but I don't want that to block this PR to land. :-)

Reply:
yep, not my intention to block it. Let me give the approval from my side.